tools/cargo_makepad/src/wasm/compile.rs

```diff
@@ -151,7 +175,16 @@ pub fn build(config:WasmConfig, args: &[String]) -> Result<WasmBuildResult, Stri
             
             cp_brotli(&dep_dir.join("src/os/web/web_gl.js"), &app_dir.join("makepad_platform/web_gl.js"), false, config.brotli)?;
             
-            cp_brotli(&dep_dir.join("src/os/web/web_worker.js"), &app_dir.join("makepad_platform/web_worker.js"), false, config.brotli)?;
+            if config.bindgen {
+                let jsfile = dep_dir.join("src/os/web/web_worker.js");
+                let js = std::fs::read_to_string(&jsfile).map_err(|e| format!("Unable to find web.js {e:?}"))?;
+                let tmp = build_dir.join("web_worker.js");
+                let mut file = std::fs::OpenOptions::new().write(true).truncate(true).create(true).open(&tmp ).unwrap();
+                file.write(format!("import init from '../bindgen.js';\n{js}").as_bytes()).unwrap();
```

Comment:
in the first version i used dynamic import, but it probably wouldn't work well with html minification. is there a better way?